README.md

```diff
@@ -427,6 +427,172 @@ way around. So
 "test": "builder run test-frontend",
 ```
 
+### Creating an Archetype
+
+Moving common tasks into an archetype is fairly straightforward and requires
+just a few tweaks to the paths defined in configuration and scripts in order
+to work correctly.
+
+#### Initializing your project
+
+An archetype is simply a standard npm module w/ a valid `package.json`. To set
+up a new archetype from scratch, make a directory for your new archetype,
+initialize NPM and link it for ease of development.
+
+```sh
+$ cd path/to/new/archetype
+$ npm init
+$ npm link
+```
+
+From your consuming project, you can now link to the archetype directly for ease
+of development after including it in your `dependencies` and creating a
+`.builderrc` as outlined above in [configuration](#configuration).
+
+```sh
+$ cd path/to/consuming/project
+$ npm link new-archetype-name
+```
+
```

Comment:
Interesting, I haven't run into any issues yet. I'll keep an eye out. Where did you find that to break?